test/CMakeLists.txt

```diff
@@ -9,3 +10,11 @@ add_executable(fisher_price fisher_price.cpp)
 # Noddy example of particle processing with GPU
 add_executable(cufisher_price cufisher_price.cu)
 target_link_libraries(cufisher_price PRIVATE CUDA::curand)
+# Unit test for atomic ops
+add_executable(test_atomic test_atomic.cu)
+target_include_directories(test_atomic PUBLIC
+  ${VecCore_INCLUDE_DIRS}
+  ${VECGEOM_INCLUDE_DIR}
+  $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/base/inc>
+  $<INSTALL_INTERFACE:base/inc>
```

Comment:
Installation directories should probably not end in `base/inc/header.h`, but just `base/header.h`.